intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -353,11 +353,12 @@ \subsection{Search for modules}
 %\ifgent{}%
     You will need to load all module(s) on any one of the lines below before the "example/1.2.3" module is available to load.
 
-      cluster/golett
-      cluster/phanpy
+      cluster/kirlia
+      cluster/joltik
```

Comment:
Actual output has these in alphabetical order, so `doduo` on top, then `joltik`, `kirlia`, ...

Reply:
I don't think `module spider example/1.2.3` has actual output :)

Reply:
For *any* `module spider` command, the clusters will be in alphabetical order, so please fix.

Reply:
which module spider command do you want me to use?